builder/azure/arm/artifact.go

```diff
@@ -243,6 +258,16 @@ func (a *Artifact) String() string {
 			}
 		}
 	}
+	if a.isPublishedToSIG() {
+		buf.WriteString(fmt.Sprintf("ManagedImageSharedImageGalleryId: %s\n", a.ManagedImageSharedImageGalleryId))
+		buf.WriteString(fmt.Sprintf("SharedImageGalleryResourceGroup: %s\n", a.State(constants.ArmManagedImageSigPublishResourceGroup).(string)))
```

Comment:
This is where the panic is happening in the artifact test for me, do you see the same panic @VladRassokhin?

Reply:
Fixed